### ISSUE TITLE
[Global Styles] Green success button's colors are not accessible

### ISSUE THREAD

**Issue by andycochran** [zurb/foundation-sites#7376]
**How can we reproduce this bug?**
Go to Global Styles Page
Use WebAIM Color Contrast Checker (http://webaim.org/resources/contrastchecker)
Enter "FFFFFF" as "foreground" color.
Enter "3adb76" as "background" color.

**What did you expect to happen?**
Contrast ratio for normal text should be 4.5:1 or higher
WCAG AA criteria should pass

**What happened instead?**
Contrast ratio is below standard.
See: http://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=3adb76

**Comment by andycochran** [comment]
The slightest change that allows AA to pass is `#238748` (keeping the same green hue and saturation, changing only the brightness).

**Comment by Owlbertz** [comment]
Using this issue for #7378, #7377, #7104, and #7103 as well to give my general opinion: 
This is one of those typical cases where you need to decide if you want to be more accessibility compliant or focus more on design. Personally, I think since F6 seems to be more focussing on accessibility itself, it might be considered to change the colors in the SCSS to fulfill the AA. While we're on it, the docs page should also be extended with a paragraph saying a few words about contrast ratios.

**Comment by zurbrandon** [comment]
Thanks Andy, I love these! I'll have to checkout and see how your recommendations look, but i really appreciate you going through the work of finding options for us.

**Comment by andycochran** [comment]
@zurbrandon You're very welcome. I really appreciate everything Zurb does. Do you have any thoughts on the related issue of disabled buttons (#7379)?

**Comment by andycochran** [comment]
It should also be noted that this doesn't just apply to buttons. Stuff like `.label` and `.badge` are also inaccessible.

**Comment by gakimball** [comment]
I'm collecting all of the contrast issues into one: #7387
